### PR TITLE
[C-839] Fix playTrack test

### PR DESCRIPTION
**Pull request by sliptype**
### Description

Adds a couple things to allow the playTrack probers test to pass

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-c-839-fix-probers

File: packages/web/src/services/audio-player/AudioPlayer.ts [hunk 1]
```diff
@@ -212,6 +212,10 @@ export class AudioPlayer {
       this.stop()
       const prevVolume = this.audio.volume
       this.audio = new Audio()
+
+      // Connect this.audio to the window so that 3P's can interact with it.
+      window.audio = this.audio
```
Comment (amendelsohn) [inline packages/web/src/services/audio-player/AudioPlayer.ts:217]: 🤣

**Review by amendelsohn — APPROVED** [review]
🎉 Thank youuuuu!!!

**Review by dylanjeffers — APPROVED** [review]
lgtm

**Pull request merged**